src/iconsManifest/supportedExtensions.ts

```diff
@@ -4368,6 +4368,12 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'tree',
+      extensions: ['.tree'],
+      filename: true,
```

Comment:
@paulrberg thanks for this. My guess is that you didn't want this to be a filename extension. If you want to support `*.tree` files, remove the `filename: true` line and the dot from the `.tree` extension.

Reply:
Happy to help!

And thanks for your feedback. You're right, I wanted to support `*.tree` files. Fixed in the latest commit.

Reply:
Thanks @paulrberg, could you rebase this PR?

Reply:
Sure, just did it.